static/js/components/CandidateAnnotationsList.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import PropTypes from "prop-types";
+
+import { makeStyles } from "@material-ui/core/styles";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import Collapse from "@material-ui/core/Collapse";
+import ExpandLess from "@material-ui/icons/ExpandLess";
+import ExpandMore from "@material-ui/icons/ExpandMore";
+import Divider from "@material-ui/core/Divider";
+
+import * as candidatesActions from "../ducks/candidates";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+    background: theme.palette.background.paper,
+    padding: theme.spacing(1),
+    maxHeight: "15rem",
+    overflowY: "scroll",
+    // Prevent disabled annotations from being selected, but display normally for readability
+    "& .Mui-disabled": {
+      opacity: 1,
+    },
+  },
+  nested: {
+    paddingLeft: theme.spacing(4),
+    paddingTop: 0,
+    paddingBottom: 0,
+  },
+}));
+
+export const getAnnotationValueString = (value) => {
+  let valueString;
+  const valueType = typeof value;
+  switch (valueType) {
+    case "number":
+      valueString = value.toFixed(6);
+      break;
+    case "string":
+      valueString = `${value.substring(0, 15)}...`;
+      break;
+    default:
+      valueString = value.toString();
+  }
+  return valueString;
+};
+
+const CandidateAnnotationsList = ({ annotations }) => {
+  const classes = useStyles();
+
+  const dispatch = useDispatch();
+
+  const initState = {};
+  annotations.forEach((annotation) => {
+    initState[annotation.origin] = true;
+  });
+  const [openedOrigins, setopenedOrigins] = useState(initState);
+
+  const selectedAnnotationItem = useSelector(
+    (state) => state.candidates.selectedAnnotationItem
```

Comment:
```suggestion
  const selectedAnnotationKey = useSelector(
    (state) => state.candidates.selectedAnnotationKey
```
or `DataKey` or something else more explicit